packages/core/src/components/Drawer/Drawer.styles.tsx

```diff
@@ -10,7 +11,9 @@ export const { staticClasses, useClasses } = createClasses("HvDrawer", {
     overflow: "auto",
     boxShadow: theme.colors.shadow,
   },
-  background: {},
+  background: {
+    background: hexToRgbA(theme.colors.atmo4, 0.8),
```

Comment:
This `hexToRgbA` doesn't work with CSS variables, right? No use of adding it here.